api/filters/partner.py

```diff
@@ -97,22 +97,22 @@ class PartnerFilter(filters.FilterSet):
     ordering = filters.OrderingFilter(
         fields=(
             ('organization__name', 'partner'),
-            ('contact_address__country__iso_code', 'country_en'),
-            ('contact_address__city', 'city'),
+            ('organization__entity__entityversion__entityversionaddress__country__iso_code', 'country_en'),
+            ('organization__entity__entityversion__entityversionaddress__city', 'city'),
             ('partnerships__ucl_entity', 'ucl_entity'),
             ('subject_area_ordered', 'subject_area'),
         )
     )
     continent = filters.CharFilter(
-        field_name='contact_address__country__continent__name',
+        field_name='organization__entity__entityversion__entityversionaddress__country__continent__name',
         lookup_expr='iexact',
     )
     country = filters.CharFilter(
-        field_name='contact_address__country__iso_code',
+        field_name='organization__entity__entityversion__entityversionaddress__country__iso_code',
         lookup_expr='iexact',
     )
     city = filters.CharFilter(
-        field_name='contact_address__city',
+        field_name='organization__entity__entityversion__entityversionaddress__city',
```

Comment:
1.Toutes les requêtes ci-dessus me semble pas exacte à 100%. En effet, il faut prendre la dernière version et faire une recherche sur ce champ city. En effet, la relation **entity__entityversion** est un one-to-many. 

2. Ne faudrait-il pas avoir un manager qui encapsule cette complexité de notion de dernière version ? (Ajouter une annotation à la query de base et faire un filtre dessus)

Pour la réutilisation, ça me semble plus facile. De la même manière que tu as fais pour les annotate_webiste() etc